### ISSUE TITLE
CCPA support for SPA or refresh ad units

### ISSUE THREAD

**Issue by ronzilca** [prebid/Prebid.js#5440]
## Type of issue
Feature (it's a bug, I think, but according to the code, it was planned this way).

## Description
Within CCPA states, the default _usprivacy_ string is 1YNN.
But, users can choose to opt-out and avoid using cookies. Then, the _usprivacy_ string becomes 1YYN.

The problem is when I want to refresh ad units, according to the _consentManagementUsp_, the _usprivacy_ won't update.

## Steps to reproduce
1. Enable the usp consent module:
```
                pbjs.setConfig({
                    'consentManagement' : {
                        'usp'               : {
                            'cmpApi'                    : 'iab',
                            'timeout'                   : 100
                        }
                    }
                });
```

2. Change the _usprivacy_ cookie (1YYN to opt-out).
3. Ask the __uspapi for the current status:
`__uspapi('getUSPData', 1, (a, b) => { console.log(a, b) });`
4. Refresh the ad units

### Expected results
The new bid request should contain the current _usprivacy_ string.

### Actual results
The new bid request contains the old _usprivacy_ string.

## Other information
![image](https://user-images.githubusercontent.com/2135285/86152513-f56e4b00-bb08-11ea-8b8c-a9a5dbe08843.png)

**Comment by patmmccann** [comment]
According to issue #4425 it was not planned this way:

"Because the USP API does not wait for a user response, if it was not successfully obtained before the first auction, we should try again to retrieve privacy data before each subsequent auction." 

and this indeed appears to be a bug.

**Comment by patmmccann** [comment]
should be solved by #5452
